run_e3sm.template.csh

```diff
@@ -564,6 +564,19 @@ if ( `lowercase $case_run_dir` == default ) then
   set case_run_dir = ${e3sm_simulations_dir}/${case_name}/run
 endif
 
+# Default group and permissions on NERSC can be a pain for sharing data
```

Comment:
I think this would be better in the section (which I thought made the directories accessible) for ensuring group permissions exist (line 1082). Did that not work, or could you clarify what the issue was? Was accessing the folder after building but before submission too late?
Otherwise this looks fine to me